integration/src/integration/java/pl/allegro/tech/hermes/integration/setup/HermesManagementInstance.java

```diff
@@ -129,17 +137,31 @@ private HermesAPIOperations setupOperations(CuratorFramework zookeeper) {
         }
 
         private CuratorFramework startZookeeperClient() {
-            final CuratorFramework zookeeperClient = CuratorFrameworkFactory.builder()
-                    .connectString(
-                            zkClusters.stream()
-                                    .map(ClusterInfo::getConnectionString)
-                                    .collect(Collectors.joining(","))
-                    )
-                    .retryPolicy(new ExponentialBackoffRetry(1000, 3))
-                    .build();
+            final CuratorFramework zookeeperClient = buildCuratorFramework(
+                    zkClusters.stream()
```

Comment:
You are right that here is actually a bug, we should actually create a separate client and separately for each cluster check if this structure has already been created. This only manifests itself to the MultipleKafkaTest class, because only there we use more than one zk cluster

Reply:
Thanks for the response. 

@michal494 could you please replace here 
```java
zkClusters.stream()
    .map(ClusterInfo::getConnectionString)
    .collect(Collectors.joining(","))
```
with
```java
zkClusters.get(0).getConnectionString()
```

as it was before we merged the mentioned PR?

Reply:
I made this change. Moreover, I added the code for closing connections opened by the CuratorFramework instances after the wait is over. They are not needed after this.